server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/delivery/SieveExecutor.java

```diff
@@ -154,8 +155,18 @@ private void sieveMessageEvaluate(MailAddress recipient, Mail aMail, ResourceLoc
         }
     }
 
-    protected void handleFailure(MailAddress recipient, Mail aMail, Exception ex) throws MessagingException, IOException {
+    @VisibleForTesting
+    void handleFailure(MailAddress recipient, Mail aMail, Exception ex) throws MessagingException, IOException {
         aMail.setAttribute(new Attribute(SIEVE_NOTIFICATION, AttributeValue.of(true)));
```

Comment:
why do we set the attribute `SIEVE_NOTIFICATION` the mail for which we have the failure?

Reply:
I should remove this line sorry